ui/README.md

```diff
@@ -0,0 +1,12 @@
+# Data module
+
+An Android library containing common UI components, types, and utilities.
+
+## Contents
+
+- [Compatibility](#compatibility)
+
+## Compatibility
+
+- **Minimum SDK**: 28
```

Comment:
(Hopefully _soon_™) we'll remove these from our main README.md, the files where this is set are linked in our docs. Can we remove it here?

Reply:
We'll keep it for now and remove it later in the readme update PRs.

Reply:
We'll keep this until the min version is common across all modules and the community docs are updated accordingly.